platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -96,6 +96,9 @@ beforeSettings {
 }
 ```
 
+The `buildCache.local.removeUnusedEntriesAfterDays` method is now deprecated.
+If set to a non-default value, this deprecated setting will take precedence over `Settings.caches.buildCache.setRemoveUnusedEntriesAfterDays()`.
```

Comment:
```suggestion
If this setting is configured with a non-default value, it will take precedence over `Settings.caches.buildCache.setRemoveUnusedEntriesAfterDays()`.
```